dev/docs/source/workbook.rst

```diff
@@ -290,6 +290,15 @@ it. This is a mandatory method call::
    problematic and non-deterministic. An explicit ``close()`` is now
    recommended in all XlsxWriter programs.
 
+It is possible to use ``with`` statement to control your workbook lifetime like this::
```

Comment:
Change this to:

It is also possible to use a ``with`` 
`context manager <https://docs.python.org/2/reference/compound_stmts.html#with>`_ 
to control the lifetime of the workbook like this::